srcpkgs/git-absorb/template

```diff
@@ -11,7 +11,11 @@ maintainer="icp <pangolin@vivaldi.net>"
 license="BSD-3-Clause"
 homepage="https://github.com/tummychow/git-absorb"
 distfiles="https://github.com/tummychow/git-absorb/archive/refs/tags/${version}.tar.gz"
-checksum=5646857dd764d0a486405e777b3ec4e919511abc96bd0e02e71ec9e94d151115
+checksum=630e61a6edf1e244c38145bda98a522217c34d3ab70a3399af1f29f19e7ced26
+
+pre_build() {
+	cargo add git2@0.18
```

Comment:
can we use `cargo update --precise`?

Reply:
Yes but that would require manually patching Cargo.toml first since it would be incompatible with the lockfile.

Reply:
then could we be more precise about the version added? trying to keep things static between builds

Reply:
I think it would be fine since the crate and dependencies have already moved to newer nimor versions upstream. And the rust ecosystem doesn't really have a backport culture for crates like these. So you wouldn't have `git2-v0.18.4` backported from `git2-v0.19.1`.

Reply:
Reading the manifest, looks like I can do `cargo add  git2@=0.18.3` anyway.